source/loader/layers/sanitizer/device_sanitizer_report.hpp

```diff
@@ -0,0 +1,95 @@
+/*
+ *
+ * Copyright (C) 2023 Intel Corporation
+ *
+ * Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+ * See LICENSE.TXT
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ * @file device_sanitizer_report.hpp
+ *
+ */
+
+#pragma once
+
+#include <cinttypes>
+
+namespace ur_sanitizer_layer {
+
+enum class DeviceSanitizerErrorType : int32_t {
+    UNKNOWN,
+    OUT_OF_BOUNDS,
+    MISALIGNED,
```

Comment:
same suggestion in another PR. suggest to add unified prefix for these definition.

Reply:
I've comment this on another PR.